google-cloud-tools-plugin/build.gradle

```diff
@@ -48,9 +61,19 @@ dependencies {
     testRuntime files('../google-account-plugin/lib/google-gct-login-context-ij-pg.jar')
 }
 
+task testJar(type: Jar, dependsOn: testClasses) {
+    baseName = "${project.archivesBaseName}-tests"
+    from sourceSets.test.output
+}
+
 configurations {
     // this is already obtained from google-api-java-client-min-repackaged
     compile.exclude group: 'com.google.api-client', module: 'google-api-client'
+    tests
+}
+
+artifacts {
```

Comment:
Also the 'utlimate' module also uses this tests configuration (see its build.gradle:40)